Improve scroll behavior auto description (#23103)
mdn/content: files/en-us/web/api/window/scroll/index.md

    - `left`
      - : Specifies the number of pixels along the X axis to scroll the window or element.
    - `behavior`

>       - : `auto` (default): The scrolling happens with the same value of style [scroll-behavior](/en-US/docs/Web/CSS/scroll-behavior), which is set or inherited on the scrolling element.
>       - : `smooth`:The scrolling animates smoothly.

Reviewer: This is not a DL. You need to add another UL as the subitem of this, like:

```suggestion
      - : Values include:
        - `auto` (default): The scrolling happens with the same value of style [scroll-behavior](/en-US/docs/Web/CSS/scroll-behavior), which is set or inherited on the scrolling element.
        - `smooth`: The scrolling animates smoothly.
```

 (Please don't commit this; you need to make the wording make sense by adding an overview sentence)

    - `left`
      - : Specifies the number of pixels along the X axis to scroll the window or element.
    - `behavior`

>       - : Browsers scrolling an element to a position can have the following scroll behaviors:
>       - `auto` (default): The scrolling happens with the same value of style [scroll-behavior](/en-US/docs/Web/CSS/scroll-behavior), which is set or inherited on the scrolling element.
>       - `smooth`:The scrolling animates smoothly.

Reviewer: ```suggestion
      - : Browsers scrolling an element to a position can have one of the following scroll behaviors:
      - `auto` (default): The scrolling behavior is controlled by the [`scroll-behavior`](/en-US/docs/Web/CSS/scroll-behavior) CSS style, which is set or inherited on the scrolling element.
      - `smooth`: The scrolling animates smoothly.
```
